lib/myFTClient.js

```diff
@@ -676,6 +718,17 @@ function getGroupsFollowingConcept(licenceId, conceptId) {
 }
 
 /**
+ * Gets the users that are following a concept/topic as an individual
+ * @param {String} licenceId -
+ * @param {String} conceptId -
+ * @returns {Promise} response -
+ */
+function getUsersFollowingConceptAsIndividual(licenceId, conceptId) {
```

Comment:
I made this specific to individual follows just to make it easier to read, as this function is used in other apps. Appreciate this will need me to duplicate the function for non-individual follows, but it will save developers digging in to `kat-client-proxies` to figure out how the params affect the query.

For the group follows, I was saving that for when we come to the groups stories. Or is this something needed for the "All Users" group?